tasks/gulp/copy-to-destination.js

```diff
@@ -15,54 +15,61 @@ const taskArguments = require('../task-arguments')
 
 gulp.task('copy-files', () => {
   return merge(
+    // Copy source JavaScript
     gulp.src([
-      `${configPaths.src}**/*`,
-
-      // Exclude files from copy
-      '!**/.DS_Store',
-      '!**/*.mjs',
-      '!**/*.test.*',
-      '!**/__snapshots__/',
-      '!**/__snapshots__/**',
-
-      // Preserve destination README when copying to ./package
-      // https://github.com/alphagov/govuk-frontend/tree/main/package#readme
-      `!${configPaths.src}README.md`,
-
-      // Exclude files from other streams
-      `!${configPaths.src}**/*.scss`,
-      `!${configPaths.components}**/*.yaml`
-    ]),
-
-    // Add CSS prefixes to Sass
-    gulp.src(`${configPaths.src}**/*.scss`)
-      .pipe(postcss([autoprefixer], { syntax: postcssScss })),
-
-    // Generate fixtures.json from ${component}.yaml
-    gulp.src(`${configPaths.components}**/*.yaml`, { base: configPaths.src })
-      .pipe(map((file, done) =>
-        generateFixtures(file)
-          .then((fixture) => done(null, fixture))
-          .catch(done)
-      ))
-      .pipe(rename({
-        basename: 'fixtures',
-        extname: '.json'
-      })),
-
-    // Generate macro-options.json from ${component}.yaml
-    gulp.src(`${configPaths.components}**/*.yaml`, { base: configPaths.src })
-      .pipe(map((file, done) =>
-        generateMacroOptions(file)
-          .then((macro) => done(null, macro))
-          .catch(done)
-      ))
-      .pipe(rename({
-        basename: 'macro-options',
-        extname: '.json'
-      }))
+      `${configPaths.src}**/*.mjs`,
+      `!${configPaths.src}**/*.test.*`
+    ]).pipe(gulp.dest(`${taskArguments.destination}/govuk-esm/`)),
+
+    merge(
+      gulp.src([
+        `${configPaths.src}**/*`,
+
+        // Exclude files from copy
+        '!**/.DS_Store',
+        '!**/*.mjs',
+        '!**/*.test.*',
+        '!**/__snapshots__/',
+        '!**/__snapshots__/**',
+
+        // Preserve destination README when copying to ./package
+        // https://github.com/alphagov/govuk-frontend/tree/main/package#readme
+        `!${configPaths.src}README.md`,
+
+        // Exclude files from other streams
```

Comment:
```suggestion
        // Exclude files from other streams (MJS and test files are already excluded a bit above)
```

Reply:
@romaricpascal This one definitely needed a better comment

It was actually excluding files handled by the PostCSS and JSON (from YAML) streams a bit _below_

Hopefully this is clearer:

```mjs
// Exclude Sass files handled by PostCSS stream below
`!${configPaths.src}**/*.scss`,

// Exclude source YAML handled by JSON streams below
`!${configPaths.components}**/*.yaml`
```